docs/evaluate/how-it-works.mdx

```diff
@@ -0,0 +1,163 @@
+
+![Describing a Temporal Application - How It Works](/img/how-it-works/htw_application.png)
+
+## Temporal Service 
+
+Temporal has two main parts:
+1. Your application (with Workflows and Activities)
+2. The Temporal Service (a set of services and components)
+
+At the heart of Temporal's architecture is the Temporal Service, which provides durability, scalability, and reliability for your application.
+It communicates with the application, manages tasks, and maintains a detailed history of each event, which it durably persists to a database.
+The Temporal Service manages critical tasks like making an API call or updating a database, by making sure they get executed, and then recording their completion.
+
+One of the biggest advantages of the Temporal Service is how it handles failures. 
+The Temporal Service maintains a meticulous log of every step in your Workflows. 
+By keeping a history of every step in your Workflow, it ensures that even if something goes wrong — your Workflow can continue from the last successful point. 
+The Temporal Service knows exactly where to resume without losing any work. 
+This saves you from having to write complex error-handling code or recovery mechanisms yourself. 
+
+You can run the Temporal Service on your own infrastructure or use Temporal Cloud, a managed service that handles operational overhead and offers scalability and expert support.
+
+![Describing a Temporal Application and Temporal Service - How It Works](/img/how-it-works/htw_application_service_server.png)
+
+
+## Workers
+The real strength of Temporal comes from the combination of your application and the Temporal Service. 
+A Worker is a component within your application and provided by the Temporal SDK. 
+Workers execute the actual tasks defined in your Workflows. The Temporal Service tracks and coordinates these Workers. 
+
+Whenever your application needs to perform a task, like sending a notification or calling an processing a payment, the Temporal Service steps in. 
+This process creates a clear, organized list of what needs to be done. 
+It hands off tasks to these Workers, making sure they’re done correctly and efficiently while it keeps track of their progress. 
+Workers which are part of your application then carry out the tasks and run your code. 
+This collaboration is crucial for building reliable, scalable, and durable applications. 
+You can run multiple Workers — often dozens, hundreds, or even thousands — to improve application performance and scalability. 
+
+![Describing a Temporal Application and Temporal Service and Workers - How It Works](/img/how-it-works/htw_application_service_workers.png)
+
+:::info 
+For a deep dive into how Temporal Service works, visit the [Temporal Encyclopedia page](/clusters).
+:::
+
+
+## Recovering from Failure
```

Comment:
I feel like this section should be moved up before you introduce the components of Temporal. This answers the question of 'why temporal' before getting into the nitty gritty.

Reply:
This is supposed to be more of a summary where all the Temporal terms are now introduced so it's explaining Temporal with our terminology that they should now know.